vmm/src/default_syscalls/x86_64.rs

```diff
@@ -324,10 +263,7 @@ pub fn default_context() -> Result<SeccompFilterContext, Error> {
                     ],
                 ),
             ),
-            (
-                libc::SYS_munmap,
-                (0, vec![SeccompRule::new(vec![], SeccompAction::Allow)]),
-            ),
+            allow_syscall(libc::SYS_munmap),
             (
                 libc::SYS_open,
```

Comment:
The filter for the open system call seems broken. As far as I understand it, any open call will match the first rule and all the other flags checking doesn't matter.

This is not an issue in your PR, but since you are here, you might want to fix or at least remove the unused code.

Reply:
I've opened #972.

Reply:
Nice catch, thanks! The rules following `vec![]` are indeed pointless.